### ISSUE TITLE
Could not resolve "/Users/wobsoriano/tmp/vuetify-sonner/src/..."

### ISSUE THREAD

**Issue by Muchieman7** [wobsoriano/vuetify-sonner#13]
Got this error on version 0.3.3
```
0|client   | ✘ [ERROR] Could not resolve "/Users/wobsoriano/tmp/vuetify-sonner/src/Toast.vue?vue&type=style&index=0&scoped=a2d3075d&lang.scss"
0|client   | 
0|client   |     node_modules/vuetify-sonner/dist/index.js:63:7:
0|client   |       63 │ import "/Users/wobsoriano/tmp/vuetify-sonner/src/Toast.vue?vue&typ...
0|client   |
```

**Comment by SaiCode-DEV** [comment]
Same issue

**Comment by SaiCode-DEV** [comment]
Is since the version yesterday. v 0.3.2 works fine

**Comment by wobsoriano** [comment]
Wahh wrong config on my part. Ill fix in a bit

**Comment by wobsoriano** [comment]
Published a new version, let me know if it works!

**Comment by SaiCode-DEV** [comment]
![grafik](https://github.com/wobsoriano/vuetify-sonner/assets/53049957/d6f50575-2479-4597-9473-cafe4ef29e5c)
Kind of... getting a different error.

**Comment by SaiCode-DEV** [comment]
![grafik](https://github.com/wobsoriano/vuetify-sonner/assets/53049957/95fe36ca-aab0-435b-b213-6158da8fd266)


```

webview] 12:37:22 [vite] Internal server error: [@vue/compiler-sfc] Failed to load TypeScript, which is required for resolving imported types. Please make sure "typescript" is installed as a project dependency.
[webview]
[webview] C:/Users/SaiCode/Documents/Server/node_modules/vuetify-sonner/dist/VSonner.vue
[webview] 3  |  import type { ToasterProps } from 'vue-sonner/lib/Toaster.vue'
[webview] 4  |
[webview] 5  |  type Props = Omit<ToasterProps, 'richColors' | 'theme' | 'closeButton' | 'className' | 'style'>
[webview]    |                    ^^^^^^^^^^^^
[webview] 6  |
[webview] 7  |  withDefaults(defineProps<Props>(), {
[webview]   Plugin: vite:vue
[webview]   File: C:/Users/SaiCode/Documents/Server/node_modules/vuetify-sonner/dist/VSonner.vue
[webview]       at ScriptCompileContext.error (C:\Users\SaiCode\Documents\Server\node_modules\@vue\compiler-sfc\dist\compiler-sfc.cjs.js:15742:11)
[webview]       at importSourceToScope (C:\Users\SaiCode\Documents\Server\node_modules\@vue\compiler-sfc\dist\compiler-sfc.cjs.js:18399:16)
[webview]       at resolveTypeFromImport (C:\Users\SaiCode\Documents\Server\node_modules\@vue\compiler-sfc\dist\compiler-sfc.cjs.js:18391:23)
[webview]       at innerResolveTypeReference (C:\Users\SaiCode\Documents\Server\node_modules\@vue\compiler-sfc\dist\compiler-sfc.cjs.js:18279:14)
[webview]       at resolveTypeReference (C:\Users\SaiCode\Documents\Server\node_modules\@vue\compiler-sfc\dist\compiler-sfc.cjs.js:18268:36)
[webview]       at innerResolveTypeElements (C:\Users\SaiCode\Documents\Server\node_modules\@vue\compiler-sfc\dist\compiler-sfc.cjs.js:17876:24)
[webview]       at resolveTypeElements (C:\Users\SaiCode\Documents\Server\node_modules\@vue\compiler-sfc\dist\compiler-sfc.cjs.js:17822:35)
[webview]       at resolveBuiltin (C:\Users\SaiCode\Documents\Server\node_modules\@vue\compiler-sfc\dist\compiler-sfc.cjs.js:18214:13)
[webview]       at innerResolveTypeElements (C:\Users\SaiCode\Documents\Server\node_modules\@vue\compiler-sfc\dist\compiler-sfc.cjs.js:17907:20)
[webview]       at resolveTypeElements (C:\Users\SaiCode\Documents\Server\node_modules\@vue\compiler-sfc\dist\compiler-sfc.cjs.js:17822:35)
[webview] 12:37:22 [vite] Internal server error: [@vue/compiler-sfc] Failed to load TypeScript, which is required for resolving imported types. Please make sure "typescript" is installed as a project dependency.
[webview]
[webview] C:/Users/SaiCode/Documents/Server/node_modules/vuetify-sonner/dist/Toast.vue
[webview] 3  |  import type { ToastProps } from './types'
[webview] 4  |
[webview] 5  |  withDefaults(defineProps<ToastProps>(), {
[webview]    |                           ^^^^^^^^^^
[webview] 6  |    vertical: false,
[webview] 7  |    cardActionsProps: () => ({}),
[webview]   Plugin: vite:vue
[webview]   File: C:/Users/SaiCode/Documents/Server/node_modules/vuetify-sonner/dist/Toast.vue
[webview]       at ScriptCompileContext.error (C:\Users\SaiCode\Documents\Server\node_modules\@vue\compiler-sfc\dist\compiler-sfc.cjs.js:15742:11)
[webview]       at importSourceToScope (C:\Users\SaiCode\Documents\Server\node_modules\@vue\compiler-sfc\dist\compiler-sfc.cjs.js:18399:16)
[webview]       at resolveTypeFromImport (C:\Users\SaiCode\Documents\Server\node_modules\@vue\compiler-sfc\dist\compiler-sfc.cjs.js:18391:23)
[webview]       at innerResolveTypeReference (C:\Users\SaiCode\Documents\Server\node_modules\@vue\compiler-sfc\dist\compiler-sfc.cjs.js:18279:14)
[webview]       at resolveTypeReference (C:\Users\SaiCode\Documents\Server\node_modules\@vue\compiler-sfc\dist\compiler-sfc.cjs.js:18268:36)
[webview]       at innerResolveTypeElements (C:\Users\SaiCode\Documents\Server\node_modules\@vue\compiler-sfc\dist\compiler-sfc.cjs.js:17876:24)
[webview]       at resolveTypeElements (C:\Users\SaiCode\Documents\Server\node_modules\@vue\compiler-sfc\dist\compiler-sfc.cjs.js:17822:35)
[webview]       at resolveRuntimePropsFromType (C:\Users\SaiCode\Documents\Server\node_modules\@vue\compiler-sfc\dist\compiler-sfc.cjs.js:19336:20)
[webview]       at genRuntimePropsFromTypes (C:\Users\SaiCode\Documents\Server\node_modules\@vue\compiler-sfc\dist\compiler-sfc.cjs.js:19312:17)
[webview]       at genRuntimeProps (C:\Users\SaiCode\Documents\Server\node_modules\@vue\compiler-sfc\dist\compiler-sfc.cjs.js:19300:18)
```

**Comment by wobsoriano** [comment]
Apologies. Fixed in v0.3.5.
